pkg/util/taints/taints.go

```diff
@@ -205,9 +206,9 @@ func deleteTaints(taintsToRemove []v1.Taint, newTaints *[]v1.Taint) ([]error, bo
 	return allErrs, removed
 }
 
-// addTaints adds the newTaints list to existing ones and updates the newTaints List.
-// TODO: This needs a rewrite to take only the new values instead of appended newTaints list to be consistent.
-func addTaints(oldTaints []v1.Taint, newTaints *[]v1.Taint) bool {
+// addedTaints if there are taints in old but not in new. Would return the taints. But only return true when len(oldTains) != (len(*newTaints) + len(addedTaints))
+// this means that if only there are taints in the newTaints but not the oldTaints, return nil and true.
```

Comment:
Trying to polish the comment. Is this how it suppose to sound?

```suggestion
// addedTaints returns added taints if there are taints in oldTaints but not in newTaints.
// It returns true when len(oldTains) != (len(*newTaints) + len(addedTaints)).
// If there are taints in the newTaints but not the oldTaints, return nil and true.
```

Reply:
That's ok.